spec/fixtures/snapshots/csv_imports/coinsmart-Transactions (15).csv.json

```diff
@@ -0,0 +1,241 @@
+{
+  "name": "Test Wallet",
+  "txn_count": 324,
+  "entry_count": 361,
+  "balances": {
```

Comment:
The balances are correct but see "bad_rows", please

Reply:
Maybe we can skip balances to be parsed?

Reply:
Yeap, skipped